readme.md

```diff
@@ -261,11 +261,11 @@ Methods accepting a `$timezone` argument default to `date_default_timezone_get()
 
 ### `Faker\Provider\Image`
 
-    // Image generation provided by LoremPixel (http://lorempixel.com/)
-    imageUrl($width = 640, $height = 480) // 'http://lorempixel.com/640/480/'
-    imageUrl($width, $height, 'cats')     // 'http://lorempixel.com/800/600/cats/'
-    imageUrl($width, $height, 'cats', true, 'Faker') // 'http://lorempixel.com/800/400/cats/Faker'
-    imageUrl($width, $height, 'cats', true, 'Faker', true) // 'http://lorempixel.com/grey/800/400/cats/Faker/' Monochrome image
+    // Image generation provided by Unsplash (https://source.unsplash.com/)
+    imageUrl($width = 640, $height = 480) // 'https://source.unsplash.com/640x480/'
+    imageUrl($width, $height, 'cats')     // 'https://source.unsplash.com/800x600/?cats'
+    imageUrl($width, $height, 'cats', true, 'Faker') // 'https://source.unsplash.com/800x400/?cats,Faker'
+    imageUrl($width, $height, 'cats', true, 'Faker', true) // 'https://source.unsplash.com/800x400/?grayscale,cats,Faker' Monochrome image
```

Comment:
This doesn't always produce greyscale images in my testing.

Reply:
I know, Unsplash doesn't support specific grayscale images, this is the best it gets... other option would be to remove the `$gray` flag